### PR TITLE
networkd: enable to set IFF_ALLMULTI to network device

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

**Comment by yuwata** [comment]
lgtm.

File: man/systemd.network.xml [hunk 1]
```diff
@@ -247,6 +247,12 @@
         <listitem>
           <para> A boolean. Enables or disables the change the MULTICAST flag on the device.</para>
         </listitem>
+      </varlistentry>
+       <varlistentry>
+        <term><varname>AllMulticast=</varname></term>
+        <listitem>
+          <para> A boolean. Accept or reject all multicast frames.</para>
```
Comment (keszybz) [inline man/systemd.network.xml]: Please make this explain what this setting does, and how it differs from Multicast= above. See https://serverfault.com/questions/321245/what-is-the-difference-between-multicast-and-allmulti-ifconfig-ifconfig-flags.

File: man/systemd.network.xml [hunk 2]
```diff
@@ -247,6 +247,12 @@
         <listitem>
           <para> A boolean. Enables or disables the change the MULTICAST flag on the device.</para>
```
Comment (keszybz) [inline man/systemd.network.xml]: This does not parse. Please fix it too.

Comment (ssahani) [reply man/systemd.network.xml]: https://books.google.co.in/books?id=MTibAgAAQBAJ&pg=PT556&lpg=PT556&dq=IFF_ALLMULTI&source=bl&ots=ITFWupPIGO&sig=P7J_zzqF98bSv6aMm8qzwine_Yo&hl=en&sa=X&ved=0ahUKEwiq-J3jh7DbAhUCvrwKHR4tBTQ4ChDoAQhNMAc#v=onepage&q=IFF_ALLMULTI&f=false

Comment (keszybz) [reply man/systemd.network.xml]: Right. But the problem is that the sentence in the man page above is grammatically incorrect, and in effect it's very unclear.

Comment (ssahani) [reply man/systemd.network.xml]: please suggest

Comment (yuwata) [reply man/systemd.network.xml]: I am not native, but I think the minimal way to fix this is just omitting 'the change'.

Comment (ssahani) [reply man/systemd.network.xml]: @yuwata I pushed a new  man. Please take a look

**Comment by keszybz** [comment]
The description of Multicast= is still wrong.

**Comment by ssahani** [comment]
what it should be then?

File: man/systemd.network.xml (hunk 2 above)
Comment (yuwata) [reply man/systemd.network.xml]: Hm? It seems not updated...

**Comment by yuwata** [comment]
I made a small update for the man page, and push it on this branch.

**Pull request merged**